### ISSUE TITLE
make stations bigger as you approach them

### ISSUE THREAD

**Issue by 7yl4r** [7yl4r/the-oregon-trajectory#29]
(on the travel screen)

that way it looks like you're coming in to dock

**Comment by BrianErikson** [comment]
The perspective doesn't really seem right for this? If you're strictly traveling to the right, depth should be irrelevant. It might be interesting to provide a meter of sorts to show you how far you have to go to get to the next station, though.

**Comment by 7yl4r** [comment]
Yeah, the implication is that you'd be traveling into the screen in
addition to the right. I'm not sure how it will look, but the way i'm
imagining it I think it might be cool.

**Comment by terpin** [comment]
Could you somehow script the size of the station sprite depending on where
the player's ship is on the screen? Like, the station is only 10% of its
size when it first appears on screen, but gradually resizes as you approach
it?

**Comment by 7yl4r** [comment]
@terpin, yes! that's exactly what I had in mind. We'll keep this up as another post-v1.0.0 enhancement to try out if we have time.
